content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -0,0 +1,105 @@
+
+To enable the **Asset Processor** in ACS mode as a server, a .setreg file needs these settings:
+```
+{
+  "Amazon": {
+    "AssetProcessor": {
+      "Settings": {
+        "Server": {
+          "enableCacheServer": true,
+          "cacheServerAddress": "T:/o3de/cache_server",
+          "ACS FBX Glob": {
+            "glob": "*.fbx",
+            "checkServer":  true
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+This will enable the cache server mode for an Asset Processor where it will write out the cached archive files to `T:/o3de/cache_folder` for all FBX files.
+
+The setting key `/AssetProcessor/Settings/Server/enableCacheServer=true` enables the **Asset Processor** to run in "server mode". In this mode, the **Asset Processor** will detect source asset changes, process the product assets, archive the product files (in a ZIP archive), and save out the archive files in the remote folder.
+
+The setting key `/AssetProcessor/Settings/Server/cacheServerAddress=<remote_shared_path>` points to a remote folder that the server and all clients can read and write from, respectively. The transfer folder should already be set up before launching the **Asset Processor**.
+
+The setting key `/AssetProcessor/Settings/Server/ACS FBX Glob={}` object specifies FBX source assets as a file type to cache. There can be a number of entries specified in the settings registry where the entry’s title needs to start with the letters ACS such as "ACS Our Textures" and "ACS Audio Files". The "glob" pattern can be used to capture files by extension or some basic matching pattern. It is important to flag the entry with "checkServer" to true to enable the entry for caching.
+
+
+## Run the Other Asset Processors in ACS in Client Mode
+
+To run the **Asset Processor** in ACS mode as a client the machine will need access to the remote folder, enable the cache system in client mode, and specify the asset file types to pull from the remote server.
+
+```
+{
+  "Amazon": {
+    "AssetProcessor": {
+      "Settings": {
+        "Server": {
+          "enableCacheServer": false,
+          "cacheServerAddress": "T:/o3de/cache_server",
+          "ACS FBX Glob": {
+            "glob": "*.fbx",
+            "checkServer":  true
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The setting key `/AssetProcessor/Settings/Server/enableCacheServer=false` enables the **Asset Processor** to run in `client mode`. In this mode, the **Asset Processor** will read the source asset changes, check the remote folder for the product archives, and process the asset if it was not found. It is possible to have a hybrid of remote and local source assets since end users will add new assets locally before submitting them to a remote source asset repository.
+
+The cache server address and ACS specifier blocks should match the **Asset Cache Server**.
+
+## Configuring an ACS Block
```

Comment:
I know saying "ACS server" is redundant, but since we're referring to the actual server running AP in ACS mode, it seems appropriate.

```suggestion
The setting key `/AssetProcessor/Settings/Server/enableCacheServer=false` enables Asset Processor to run in `client mode`. In this mode, Asset Processor reads the source asset changes, checks the remote directory for the product archives, and processes the asset if it's not found. It's possible to have a hybrid of remote and local source assets since team members will add new assets locally before submitting them to a remote source asset repository.

The cache server address and ACS specifier blocks should match the ACS server.

## Configuring an ACS block
```